app/javascript/hyacinth_ui_v1/components/digital_objects/DigitalObjectList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import { has } from 'lodash';
+
+const titleForDigitalObject = (digObj) => {
```

Comment:
This function isn't necessary now that `title` is a query-able field in the DigitalObject graphql query.  This might also eliminate the need for retrieving dynamicFieldData in getChildStructureQuery, if that's the only reason you were including it.

Reply:
I removed these two functions and patched the relevant REST controller to mimic the gql query. I didn't tag the commit with the issue number, since it seems like a standalone thing.

Reply:
Good move, thanks!